Packages/Tags/Runtime/AtomTags.cs

```diff
@@ -16,6 +16,9 @@ namespace UnityAtoms.Tags
     [AddComponentMenu("Unity Atoms/Tags")]
     public sealed class AtomTags : MonoBehaviour, ISerializationCallbackReceiver
     {
+        // declaring events
+        public AtomEventString OnTagAdded;
+        public AtomEventString OnTagRemoved;
```

Comment:
Where does the `AtomEventString` type come from? I can't find it in the project.
It should be `StringEvent`, shouldn't it?

https://github.com/unity-atoms/unity-atoms/blob/canary/Packages/BaseAtoms/Runtime/Events/StringEvent.cs